src/actions/index.ts

```diff
@@ -36,6 +36,9 @@ export {
   type SimulateProveWithdrawalTransactionParameters,
   type SimulateProveWithdrawalTransactionReturnType,
 } from './public/L1/simulateProveWithdrawalTransaction.js'
+export { estimateFees, type EstimateFeesParameters } from './public/L2/estimateFees.js'
+export { estimateL1Fee, type EstimateL1FeeParameters } from './public/L2/estimateL1Fee.js'
+export { estimateL1GasUsed, type GasPriceOracleEstimator } from './public/L2/estimateL1GasUsed.js'
```

Comment:
This looks like a bug
```suggestion
export { estimateL1GasUsed, type EstimateL1GasUsedParameters } from './public/L2/estimateL1GasUsed.js'
```